homeassistant/components/octoprint/sensor.py

```diff
@@ -211,16 +211,23 @@ def native_value(self):
 
         for temp in printer.temperatures:
             if temp.name == self._api_tool:
-                return round(
+                val = (
                     temp.actual_temp
                     if self._temp_type == "actual"
-                    else temp.target_temp,
-                    2,
+                    else temp.target_temp
                 )
+                if not val:
```

Comment:
Do we need to handle a valid value `0`? Then we should check for None.

Reply:
Switched to check for None